templates/default/babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["@babel/preset-env","next/babel"]
+  "presets": [
+    [
+      "next/babel", 
+      {
+        "preset-env": { "targets": { "node": "current" } }
```

Comment:
this was required to be able to extend a class in document.js

Reply:
Fixes #127?